app/frontend/src/features/profile/view/ProfilePage.tsx

```diff
@@ -57,23 +68,47 @@ export default function ProfilePage() {
     username ?? (currentUser.data?.username || "")
   );
 
+  const [isRequesting, setIsRequesting] = useState(false);
+  const [isSuccessRequest, setIsSuccessRequest] = useState(false);
+
+  const handleClose = () => {
+    setIsSuccessRequest(false);
+  };
+
```

Comment:
Can we have a useEffect here with a dependency on is requesting, that uses `element.scrollIntoView()` if it's true? (Because tapping request on mobile appears to do nothing atm because it appears off screen)

Reply:
To scroll to the request box (I think a ref on Collapse should work)